ros_robot_controller/BusServoState[] state

float64 duration
